pkg/sfu/downtrack.go

```diff
@@ -19,6 +20,19 @@ import (
 	"github.com/livekit/livekit-server/pkg/sfu/buffer"
 )
 
+// TrackSender defines a  interface send media to remote peer
+type TrackSender interface {
+	UptrackLayersChange(availableLayers []uint16, layerAdded bool) (int32, error)
+	WriteRTP(p *buffer.ExtPacket, layer int32) error
+	Close()
+	// ID is the unique identifier for this Track. This should be unique for the
+	// stream, but doesn't have to globally unique. A common example would be 'audio' or 'video'
+	// and StreamID would be 'desktop' or 'webcam'
+	ID() string
```

Comment:
I think in our case this is the `TR_` id which is globally unique. I think we will need to this be unique in our system. This comment from `ion-sfu` may not apply.

I am relying on this id  being unique in `StreamAllocator`. At least across all streams in the subscriber peer connection. We should probably revisit the comments and make sure they are accurate at some point.

Reply:
Done